pkg/plugins/manager/client/client.go

```diff
@@ -100,12 +108,21 @@ func (s *Service) CallResource(ctx context.Context, req *backend.CallResourceReq
 		removeConnectionHeaders(req.Headers)
 		removeHopByHopHeaders(req.Headers)
 
```

Comment:
It's more about the `removeNonAllowedHeaders` being so generic feels like if people add to it, it might get missed. Maybe less of a problem since it's remove from the response, but still 🤷

Reply:
Fixed